src/Bridge/Twig/Extension/EnumExtension.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Greg0ire\Enum\Bridge\Twig\Extension;
+
+use Greg0ire\Enum\AbstractEnum;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class EnumExtension extends \Twig_Extension
+{
+    /**
+     * @var TranslatorInterface
+     */
+    private $translator;
+
+    /**
+     * @param TranslatorInterface $translator
+     */
+    public function __construct(TranslatorInterface $translator = null)
+    {
+        $this->translator = $translator;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFilters()
+    {
+        return [
+            new \Twig_SimpleFilter('enum_label', [$this, 'label']),
+        ];
+    }
+
+    /**
+     * @param mixed       $value
+     * @param string      $class             The enum class name
+     * @param string      $translationDomain The translation domain to use if the translator if available.
+     *                                       string: Use the specified one
+     *                                       null: Use the default one
+     *                                       false: Do not use the translator
+     * @param bool|string $prefixWithClass   Prefix key with class name if true.
```

Comment:
Naming this one `$separator` would maybe feel more natural.
$separator = true => I want a separator, which implies to prefix with a class
$separator = 'ø' => I want 'ø' as a separator, which implies to prefix with a class
$separator = false => I don't want any separator, because I don't even need a prefix.